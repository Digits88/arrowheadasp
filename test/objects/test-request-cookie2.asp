Test getting a cookie.

Hello = <%= Request.Cookies("Hello") %><br>
Hello("Test") = <%= Request.Cookies("Hello")("Test") %><br>
Hello("There") = <%= Request.Cookies("Hello")("There") %><br>
