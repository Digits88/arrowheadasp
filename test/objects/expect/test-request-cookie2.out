Test getting a cookie.

Hello = ^THERE=two&TEST=test|TEST=test&THERE=two$<br>
Hello("Test") = test<br>
Hello("There") = two<br>
